salt/modules/boto_iam.py

```diff
@@ -153,6 +153,297 @@ def role_exists(name, region=None, key=None, keyid=None, profile=None):
         return False
 
 
+def describe_role(name, region=None, key=None, keyid=None, profile=None):
+    '''
+    Get information for a role
+
+    CLI example::
+
+        salt myminion boto_iam.describe_role myirole
+    '''
+    conn = _get_conn(region, key, keyid, profile)
+    try:
+        info = conn.get_role(name)
+        if not info:
+            return False
+        return info
+    except boto.exception.BotoServerError as e:
+        log.debug(e)
+        msg = 'Failed to get {0} information.'
+        log.error(msg.format(name))
+        return False
+
+
+def create_user(user_name, path=None, region=None, key=None, keyid=None,
+                profile=None):
+    '''
+    Create a user
+
+    CLI example::
+
+        salt myminion boto_iam.create_user myuser
+    '''
+    if not path:
+        path = '/'
+    if __salt__['boto_iam.get_user'](user_name, region, key, keyid, profile):
+        return True
+    conn = _get_conn(region, key, keyid, profile)
+    try:
+        conn.create_user(user_name, path)
+        log.info('Created user : {0} .'.format(user_name))
+        return True
+    except boto.exception.BotoServerError as e:
+        log.debug(e)
+        msg = 'Failed to create user {0} .'
+        log.error(msg.format(user_name))
+        return False
```

Comment:
Maybe as an idea, if you want to return `False` even on crash, then at least there should be a parameter that says `suppress_errors` or so.